torchvision/datasets/utils.py

```diff
@@ -83,8 +95,6 @@ def download_url(
         md5 (str, optional): MD5 checksum of the download. If None, do not check
         max_redirect_hops (int, optional): Maximum number of redirect hops allowed
     """
-    import urllib
-
```

Comment:
`urllib` is part of the standard library so there is no reason to import it lazily.